source/ht_file.cpp

```diff
@@ -183,15 +162,12 @@ namespace Hatchit
         **/
         bool File::Close(void)
         {
-            int ret = 0;
-            //Close file if handle is active
-            if (m_handle)
+            if (m_handle.is_open())
             {
-                ret = fclose(m_handle);
-                m_handle = nullptr;
+                m_handle.close();
+                return true;
             }
-
-            return (ret <  0) ? false : true;
+            return false;
```

Comment:
Shouldn't return false if we had an empty handle in the first place.  If m_handle.close() returns some sort of error code, use those results to determine whether we were successful or not.